cmd/osm-bootstrap/crds/config_mesh_root_certificate.yaml

```diff
@@ -39,10 +39,6 @@ spec:
           jsonPath: .status.currentState
           name: State
```

Comment:
@jaellio  Does this name need to be changed at all?

Reply:
Yes, this should be updated to ProviderState